src/python/director/motionplanningpanel.py

```diff
@@ -82,7 +82,7 @@ def __init__(self, robotStateModel, robotStateJointController, teleopRobotModel,
         # Foot step planning
         self.placer = None
         self.ui.walkingPlanButton.connect('clicked()', self.onWalkingPlan)
-        
+        self.ui.walkingFakeExeButton.connect('clicked()', self.onWalkingFakeExecute)
```

Comment:
i understand, please change to teleportRobotToStanceFrame can see if you can make it identical to the version in tabledemo.py and valvedemo.py

where we have common functions like this we often move them to robottasks.py